recipe/build.sh

```diff
@@ -40,7 +40,7 @@ cmake \
     ${CMAKE_ARGS} \
     -D CMAKE_BUILD_TYPE=${BUILD_TYPE} \
     -D CMAKE_INSTALL_PREFIX=$PREFIX \
-    -D CMAKE_INSTALL_LIBDIR:PATH=$PREFIX/lib \
+    -D CMAKE_INSTALL_LIBDIR=lib \
```

Comment:
This is also the default according to the tables in:
https://cmake.org/cmake/help/latest/command/install.html#signatures
So we could probably just remove it.

Reply:
This specification is actually preferred for conda-forge recipes ([ref](https://conda-forge.org/docs/maintainer/knowledge_base.html)), because sometimes `LIBDIR` is `lib64`

Reply:
Ah, @mwtoews, thanks for pointing to that documentation.  Very good to know!